mail/templates/email_base.html

```diff
@@ -277,9 +275,9 @@
                         <br><br>
                         {% endif %}
                         {% endblock %}
-                        MIT Office of Digital Learning, 600 Technology Square, 2nd Floor, Cambridge, MA 02139
+                        MIT Open Learning, 600 Technology Square, 2nd Floor, Cambridge, MA 02139
                         <br>
-                        &copy; {% now "Y" %} {{ site_name }}, All Rights Reserved. <a href="{{ base_url }}{% url 'terms-and-conditions' %}" style="color: #b0b0b0; text-decoration: underline;">Terms and Conditions and Privacy Policy</a>
+                        &copy; {% now "Y" %} {{ site_name }}, All Rights Reserved. <a href="{{ base_url }}/terms-of-service" style="color: #b0b0b0; text-decoration: underline;">Terms and Conditions</a> and <a href="{{ base_url }}/privacy-policy" style="color: #b0b0b0; text-decoration: underline;">Privacy Policy</a>
```

Comment:
```suggestion
                        &copy; {% now "Y" %} {{ site_name }}, All Rights Reserved. <a href="{{ base_url }}/terms-of-service" style="color: #b0b0b0; text-decoration: underline;">Terms of Service Conditions</a> and <a href="{{ base_url }}/privacy-policy" style="color: #b0b0b0; text-decoration: underline;">Privacy Policy</a>
```

Link text should match the URL & page title.